pkg/util/node/node.go

```diff
@@ -183,3 +183,48 @@ func PatchNodeStatus(c clientset.Interface, nodeName types.NodeName, oldNode *v1
 	}
 	return updatedNode, nil
 }
+
+func UpdateNodeStatus(c clientset.Interface, node *v1.Node, attachedVolumes []v1.AttachedVolume) (error, bool) {
```

Comment:
There should be a few calls sites in the kubelet that can use this as well - check `kubelet_node_status.go`